### PR TITLE
Update menu + footer

### PR DESCRIPTION
**Pull request by EmmanuelDrouin**
- Rename "Technology" by "Products"
- Update "Products" sub-menu (SDK, Services, Process, Liteflow)
- Remove link to the Get started and The marketplace
- Remove link to the Token page
- remove link to the Enterprise page
- Update Roadmap link to the [github roadmap](https://github.com/orgs/mesg-foundation/projects/25)
- Update copyright date (2020)
- Remove Contributions link from the menu (still in the footer)
- Remove documentation link
- Remove tutorials link
- Update footer navigation organization

### CODE REVIEW

**Comment by EmmanuelDrouin** [comment]
@antho1404 i had the same issue then before with external link at the first level of the menu. It's not working (github link). Can you check why?

File: store/index.js [hunk 1]
```diff
@@ -1251,30 +1246,17 @@ export const getters = {
...
-        { text: 'Tutorials', to: externalLinks.tutorialsMenu },
-        { text: 'Github', to: externalLinks.github }
+        { text: 'Liteflow', to: externalLinks.liteflowMenu }
       ]
     },
     { text: 'Showcase', to: links.showcase },
-    { text: 'Contributions', to: links.contributions },
-    { text: 'Token', to: links.token },
-    { text: 'Get started', to: externalLinks.quickStartCTAMenu, type: 'button' }
+    { text: 'Github', to: externalLinks.github }
```
Comment (antho1404) [inline store/index.js:1259]:
```suggestion
    { text: 'Github', href: externalLinks.github }
```

**Comment by antho1404** [comment]
menu fixed

**Comment by EmmanuelDrouin** [comment]
@antho1404 can you add a target="_blank" for external link in the menu? We have it for the Liteflow link (in the products sub-menu) but not for the Github link.

**Comment by antho1404** [comment]
target=_blank added ;)

**Pull request merged**